workflows/energyplus.py

```diff
@@ -203,281 +190,306 @@ def run_energyplus(self, isIP, run_directory, file_name, args, by_api):
 
         v = Version()
         is_found, current_version, numeric_version = v.check_energyplus_version(full_file_path)
-        if is_found:
-            if by_api:
-                # if calling through API, then the arguments don't include the E+ binary
-                command_line_args = []
-            else:
-                command_line_args = [energyplus_binary]
-
-            if extension == '.imf':
-                command_line_args += ['--epmacro']
-
-            # should be able to do this once we get pyExpandObjects going
-            if extension != '.epJSON':
-                command_line_args += ['--expandobjects']
-
-            # TODO: add -d run_directory support to the E+ call here
+        if not is_found:
```

Comment:
Reversed this `if` condition so that we could leave now if it isn't found, and it makes for less indentation in the rest of the function.